health-services/project-factory/src/server/validators/campaignValidators.ts

```
@@ -495,36 +479,38 @@
 
 
 async function validateCampaignId(request: any) {
-    const { campaignId, tenantId, type } = request?.body?.ResourceDetails;
+    const { campaignId, tenantId, type, additionalDetails } = request?.body?.ResourceDetails;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)